easybuild/easyconfigs/p/Portcullis/Portcullis-1.2.2-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,90 @@
+easyblock = 'Bundle'
+
+name = 'Portcullis'
+version = '1.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://github.com/maplesond/portcullis"
+description = """Portcullis stands for PORTable CULLing of Invalid Splice junctions from
+pre-aligned RNA-seq data. It is known that RNAseq mapping tools generate many
+invalid junction predictions, particularly in deep datasets with high coverage
+over splice sites. In order to address this, instead for creating a new RNAseq
+mapper, with a focus on SJ accuracy we created a tool that takes in a BAM file
+generated by an RNAseq mapper of the user's own choice (e.g. Tophat2, Gsnap,
+STAR2 or HISAT2) as input (i.e. it's portable). It then, analyses and quantifies
+all splice junctions in the file before, filtering (culling) those which are
+unlikely to be genuine. Portcullis output's junctions in a variety of formats
+making it suitable for downstream analysis (such as differential splicing
+analysis and gene modelling) without additional work. Portcullis can also filter
+the original BAM file removing alignments associated with bad junctions."""
+github_account = 'maplesond'
```

Comment:
@lexming Looks generally fine but I'm not sure why this `github_account` is here given you don't refer to it elsewhere...

Reply:
@verdurin you are right, I was using `GITHUB_SOURCE` in an earlier version